tests/php/src/REST/AdminOnboardingControllerTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace WeDevs\Dokan\Test\REST;
+
+use WeDevs\Dokan\Admin\SetupWizard;
+use WeDevs\Dokan\REST\AdminOnboardingController;
+use WeDevs\Dokan\Test\DokanTestCase;
+use WP_REST_Request;
+use WP_REST_Server;
+
+class AdminOnboardingControllerTest extends DokanTestCase {
+	/**
+	 * @var WP_REST_Server
+	 */
+	protected $server;
+
+	/**
+	 * @var string
+	 */
+	protected $namespace = 'dokan/v1';
+
+	/**
+	 * @var string
+	 */
+	protected $rest_base = 'admin-onboarding';
+
+	/**
+	 * @var int
+	 */
+	protected $admin_id;
+
+	/**
+	 * Setup test environment
+	 *
+	 * @return void
+	 */
+	public function setUp(): void {
+		parent::setUp();
+
+		global $wp_rest_server;
+        $wp_rest_server = new WP_REST_Server();
+        $this->server   = $wp_rest_server;
+        do_action( 'rest_api_init' );
+
+		// Create an admin user
+		$this->admin_id = self::factory()->user->create(
+			[
+				'role' => 'administrator',
+			]
+		);
+
+		// Add capabilities to admin user
+		$admin = get_user_by( 'id', $this->admin_id );
+		$admin->add_cap( 'manage_woocommerce' );
+	}
+
+	/**
+	 * Test if the endpoint exists
+	 *
+	 * @return void
```

Comment:
It is best practice to use `@test` in comment along with the method prefix `test_`.